Styles/styles.css

```diff
@@ -0,0 +1,233 @@
+@import url('https://fonts.googleapis.com/css?family=Roboto+Mono');
+
+/************
+*Site Header*
+*************/
+
+* {
+    margin: 0;
+    padding: 0;
+}
+
+body {
+  background: white;
+}
+
+header {
+  position: fixed;
+  background: #fff;
```

Comment:
Also, you should very rarely need to repeat yourself with regards to color in the same section (site header, for example). You don't always have to include the body for the header - you could just target the header itself to control its color. That way your code stays DRY and it's easier to read and make changes.